Sources/PointFree/SiteMiddleware.swift

```diff
@@ -18,12 +19,15 @@ public let siteMiddleware: Middleware<StatusLineOpen, ResponseEnded, Prelude.Uni
       protect: isProtected
     )
     <| currentUserMiddleware
+    >-> currentSubscriptionMiddleware
     >-> render(conn:)
 
-private func render(conn: Conn<StatusLineOpen, T2<Database.User?, Route>>)
+private func render(conn: Conn<StatusLineOpen, T3<Database.Subscription?, Database.User?, Route>>)
   -> IO<Conn<ResponseEnded, Data>> {
 
-    let (user, route) = (conn.data.first, conn.data.second)
+    let (subscription, user, route) = (conn.data.first, conn.data.second.first, conn.data.second.second)
```

Comment:
really starting to feel we should bring `RequestContext` back! These 3 values should prob go into every page's middleware...

Reply:
It's possible! Thought here they can be optional and we `require` them a lot of places in middleware, so not sure how we resolve both worlds yet.